File: img_load/src/main/java/com/ad/img_load/pickerview/JsonUtils.java
package com.ad.img_load.pickerview;

import android.content.Context;
import android.content.res.AssetManager;


import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

/**
 * Company:
 * FileName:JsonUtils
 * Author:android
 * Mail:2898682029@qq.com
 * Date:20-2-20 下午1:41
 * Description:${DESCRIPTION}
 */
public class JsonUtils {

    public static String getJson(String fileName,Context context) {
        StringBuilder stringBuilder = new StringBuilder();
        try {
            AssetManager assetManager = context.getAssets();
            BufferedReader bf = new BufferedReader(new InputStreamReader(
                    assetManager.open(fileName)));
            String line;
            while ((line = bf.readLine()) != null) {
                stringBuilder.append(line);
            }
        } catch (IOException e) {
            e.printStackTrace();
        }
        return stringBuilder.toString();
    }



}

File: net_work/src/main/java/com/comm/net_work/gson/JsonUtils.kt
package com.comm.net_work.gson

import android.app.Application
import java.io.BufferedReader
import java.io.IOException
import java.io.InputStreamReader

/**
 * Company:AD
 * ProjectName: 无界
 * Package: com.comm.net_work.gson
 * ClassName: JsonUtils
 * Author:ShiMing Shi
 * CreateDate: 2022/9/4 15:39
 * Email:shiming024@163.com
 * Description:
 */
object JsonUtils {

    fun getJson(context: Application,fileName: String) : String{
        val stringBuilder = StringBuilder()
        try {
            val assetManager = context.assets
            val bf = BufferedReader(
                InputStreamReader(
                    assetManager.open(fileName)
                )
            )
            var line: String?
            while (bf.readLine().also { line = it } != null) {
                stringBuilder.append(line)
            }
        } catch (e: IOException) {
            e.printStackTrace()
        }
        return stringBuilder.toString()
    }
}